src/ServiceContainer/SymfonyExtension.php

```diff
@@ -110,6 +112,15 @@ public function configure(ArrayNodeDefinition $builder): void
      */
     public function load(ContainerBuilder $container, array $config): void
     {
+        if (null !== $config['env_file']) {
+            $this->loadEnvVars($container, $config['env_file']);
+
+            $environment = getenv('APP_ENV');
+            $config['kernel']['env'] = $environment ?? 'test';
+        } else {
```

Comment:
If we don't remove this method call from `loadKernel` we can remove this `if` instead.

Reply:
I am not sure I understand. We need this else because the method below raise an exception.

Reply:
https://github.com/FriendsOfBehat/SymfonyExtension/blob/db6193144e1f738a2e5ec201d1937c94db95259d/src/ServiceContainer/SymfonyExtension.php#L286-L288

It does not throw an exception if bootstrap file is not defined.